bridge/client/webrtc.js

```diff
@@ -1656,6 +1656,18 @@
        checkArguments("createObjectURL", "Blob", 1, arguments);
     };
 
+    Object.defineProperty(HTMLVideoElement.prototype, "srcObject", {
+        "get": function () {
+            console.log("getting video element's srcObject");
+            return this._srcObject;
+        },
+        "set":  function (stream) {
+            console.log("setting video element's srcObject to: " + stream);
+            this._srcObject = stream;
+            this.src = url.createObjectURL(stream);
+        }
+    }); 
+
```

Comment:
This looks good. Please remove the comments and the double white space after "set": then we're good to go.